lib/compiler.js

```diff
@@ -65,6 +65,13 @@ exports.compile = function (code, options) {
     });
 
     importExportVisitor.finalizeHoisting();
+
+    if (getOption(options, "wrapWithRun")) {
+      result.code = "module.run(function(){" +
+        magicString.toString() + "//*/\n});";
+
+      return result;
+    }
```

Comment:
> The only reason `wrapWithRun` exists is because of the REPL so I don't think the option should exist.

Perhaps the compiler should be generally aware of whether it's compiling for the REPL, and do the right thing here (and in other cases). I agree the `wrapWithRun` option smells like an implementation detail, now that we're only toggling it in the REPL, but I'm still pretty convinced we should not wrap REPL code with `module.run(function(){...})`.